environment.yml

```diff
@@ -0,0 +1,7 @@
+channels:
```

Comment:
What's the difference between this file and `.github/environment.yml`?

Reply:
Ah I accidentally `ls`'d in the root directory and thought we didn't have one yet. Whoops. Updated now.